src/styles/header.scss

```diff
@@ -0,0 +1,69 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
```

Comment:
Only `box-sizing` is allowed in `*` selector
For reset default browser styles use tag selector